### PR TITLE
Copy resources directly without zip

### PR DESCRIPTION
**Pull request by vChavezB**
Instead of using zip files, use the source files in the resources directory. This allows to track differences easily over git as zip files cannot be compared between commits easily.

In specific I did this as I am testing the swrl rules xslt files (see #652 ) and found out that the zip file is used instead of the `src/main/resources/lode` folder.

### CODE REVIEW

**Comment by dgarijo** [comment]
@vChavezB thanks! Can you please merge in `develop` It's the branch where the latest changes are

**Comment by vChavezB** [comment]
I have changed it to the develop branch. Should I also delete the webvowl patched zip and display the source files ? At the moment I tested my changes only with the resources from the folder loder and oops.

**Comment by dgarijo** [comment]
This looks good to me. I'll have to check if those `getAbsolutePaths` do are not problematic in unix.
Thanks! Please add yourself as well in the contributor list in the README file

**Comment by dgarijo** [comment]
Ouch, no, it does not pass the tests. I will leave a few comments where the errors may be

File: src/main/java/widoco/Configuration.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public Configuration() {
 			// create a temporal folder with all LODE resources
 			tmpFolder = new File("tmp" + new Date().getTime());
 			tmpFolder.mkdir();
-			WidocoUtils.unZipIt(Constants.LODE_RESOURCES, tmpFolder.getName());
+			WidocoUtils.copyResourceDir(Constants.LODE_PATH, tmpFolder);
```
Comment (dgarijo) [inline src/main/java/widoco/Configuration.java:137]: the getName is giving errors

File: src/main/java/widoco/CreateOOPSEvalInThread.java [hunk 2]
```diff
@@ -94,7 +94,7 @@ public void run() {
                             if (!evalFolder.exists())
                                     evalFolder.mkdir();
                             evalResourcesFolder.mkdir();
-                            WidocoUtils.unZipIt(Constants.OOPS_RESOURCES, evalResourcesFolder.getAbsolutePath());
+                            WidocoUtils.copyResourceDir(Constants.OOPS_PATH, evalResourcesFolder);
```
Comment (dgarijo) [inline src/main/java/widoco/CreateOOPSEvalInThread.java:97]: see that the getAbsolutePath is needed in some versions of Java

File: src/main/java/widoco/CreateResources.java [hunk 3]
```diff
@@ -484,7 +484,7 @@ private static void createFolderStructure(String s, Configuration c, Properties
 		if (c.isCreateWebVowlVisualization()) {
 			File webvowl = new File(s + File.separator + "webvowl");
 			webvowl.mkdir();
-			WidocoUtils.unZipIt(Constants.WEBVOWL_RESOURCES, webvowl.getAbsolutePath());
```
Comment (dgarijo) [inline src/main/java/widoco/CreateResources.java:487]: same as above

**Comment by vChavezB** [comment]
Thanks for the feedback. I think I did not push the changes for those files correctly. Will have a look again ;)

**Comment by dgarijo** [comment]
Still not :(

**Comment by vChavezB** [comment]
Sorry for the multiple changes. I must admit that Java is not my first language.

I will do a clean git clone and compile again to debug this.

**Comment by dgarijo** [comment]
No rush

**Comment by vChavezB** [comment]
Ok I think it should now compile, for some reason the previous commit did not use the `copyResourceDir` method.

**Comment by dgarijo** [comment]
Thanks, looking good. Will review and merge soon!

**Review by dgarijo — APPROVED** [review]

**Pull request merged**